JSDemos/Demos/DataGrid/RecordPaging/description.md

```diff
@@ -1 +1,12 @@
-The **DataGrid** widget can automatically split its content across multiple pages and allow end users to&nbsp;navigate records using its built-in pager. This demo illustrates how you can change pager settings by&nbsp;using both the **paging** and **pager** configuration options.
\ No newline at end of file
+The **DataGrid** splits records into multiple pages. This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
+
+The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and contains the following elements:
+
+- **Page navigator**        
+Enables page navigation.
+
+- **Page size selector**        
+Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
```

Comment:
_You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and..._

Is allowedPageSizes a method, an option, or something else? I think you should specify.

Reply:
It is an option but its name, as well as that of `pageSize`, is used in this sentence as plain text: "You can also define the allowed page sizes and specify the initial page size."

Reply:
In that case, can we spell it out as "allowed page size" (use the singular form of "size"), like in your suggestion?

Reply:
No, because this is an option name.